src/shared/components/timer/timer.component.js

```diff
@@ -0,0 +1,123 @@
+import React, { PureComponent } from 'react';
+import moment from 'moment';
+import firebase from 'firebase';
+
+import { StartButton, SaveButton, Time, HistoryContainer } from './timer.styles';
+
+require('firebase/firestore');
+
+export class Timer extends PureComponent {
+  state = {
+    counting: false,
+    time: 0,
+    databaseTimes: [],
+  };
+
+  componentDidMount() {
+    this.db = firebase.firestore();
+    const times = [];
```

Comment:
korzystaj ze składni `async/await`

```suggestion
    async componentDidMount() {
       this.db = firebase.firestore();
       const times = [];
       try {
          const { docs } = this.db.collection('times').get();
          // handle docs here
       } catch (error) {
           // handle error here
       }
       const { docs } = await this.db.collection('times').get()
    }
```